backend/src/agent/persistence/client/workflow_client.go

```diff
@@ -27,11 +27,11 @@ type WorkflowClientInterface interface {
 
 // WorkflowClient is a client to call the Workflow API.
 type WorkflowClient struct {
-	informer v1alpha1.WorkflowInformer
+	informer v1beta1.PipelineRunInformer
```

Comment:
Again a comment mentioning the switch, and we will come back and define a common object